DESCRIPTION

```diff
@@ -23,12 +23,16 @@ Imports:
     dplyr,
     tibble,
     purrr,
-    glue,
     ghql,
     crul,
     readr,
     rlang,
-    fs
+    remotes,
+    fs,
+    gh,
+    gert,
+    sys,
```

Comment:
sys and withr could disappear later based on whether gert supports cloning at depth 1